operator/README.md

```diff
@@ -0,0 +1,220 @@
+# rapidast-operator
+The RapiDAST operator makes it possible to configure RapiDAST scans from your OpenShift or Kubernetes cluster. This allows you to scan your APIs from both inside and outside of your cluster.
+
+## Custom Resource Definitions
+The RapiDAST operator extends the RapiDAST custom resource definition (CRD) 
+
+## Installing the Operator
+
+There are multiple ways to install the operator onto your cluster. Read through, and select the method that best suits your situation.
+
+### Install via Operator Lifecycle Manager (OLM)
+If your cluster has OLM installed, you can use it to install and manage the RapiDAST operator. You can follow the installation instructions [here](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/install/install.md). If you're running on an OpenShift cluster, you likely have this installed by default.
+
+Install the operator by adding the following CatalogSource and Subscription resources to your cluster.
+#### Add CatalogSource
+Create a catalog source in a file `catalogsource.yaml` with the following contents. Update the value for namespace as required.
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: rapidast-catalog
+  namespace: rapidast
+spec:
+  sourceType: grpc
+  image: quay.io/redhatproductsecurity/rapidast-operator-catalog:v0.0.1
+```
+Apply the resource to your cluster
+```bash
+kubectl apply -f catalogsource.yaml
+```
+
+#### Add Subscription
+Create a subscription in a file `subscription.yaml` with the following contents, updating the namespace to match the configuration of your cluster.
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: rapidast-operator
+spec:
+  channel: alpha
+  installPlanApproval: Automatic
+  name: rapidast-operator
+  source: rapidast-catalog
+  sourceNamespace: rapidast
+```
+Now apply the subscription to your cluster 
+```bash
+kubectl apply -f subscription.yaml
+```
+
+### Install with operator-sdk
+
+You can install the operator using the Operator SDK CLI. Installation instructions are available [here](https://sdk.operatorframework.io/docs/installation/)
+
+If not already installed, install OLM
+
+```bash
+operator-sdk olm install
+```
+
+```bash
+operator-sdk run bundle quay.io/redhatproductsecurity/rapidast-operator-bundle:v0.0.1
+```
+
+You can uninstall with
+```bash
+operator-sdk cleanup --delete-all rapidast-operator
+```
+
+### Install with Makefile
+
+There is a target in the included Makefile that allows for simple deployment. This will deploy the version and image specified in the Makefile.
```

Comment:
it'd be nice to add how we can tell it's deployed properly. e.g. checking something in the 'rapidast-operator-system' namespace?